Plugin/CppApi/source/CoordinateConversionController.cpp

```diff
@@ -548,23 +617,28 @@ void CoordinateConversionController::setRunConversion(bool runConversion)
  */
 
 /*!
-  \fn void CoordinateConversionController::inputGarsConversionModeChanged();
-  \brief Signal emitted when the \l inputGarsConversionMode property changes.
+  \fn void CoordinateConversionController::runConversionChanged();
+  \brief Signal emitted when the \l runConversion property changes.
+ */
+
+/*!
+  \fn void CoordinateConversionController::pointToConvertChanged();
+  \brief Signal emitted when the \l pointToConvert property changes.
  */
 
 /*!
-  \fn void CoordinateConversionController::inputMgrsConversionModeChanged();
-  \brief Signal emitted when the \l inputMgrsConversionMode property changes.
+  \fn void CoordinateConversionController::coordinateFormatsChanged();
+  \brief Signal emitted when the list of \l coordinateFormats property changes.
  */
 
 /*!
-  \fn void CoordinateConversionController::inputUtmConversionModeChanged();
-  \brief Signal emitted when the \l inputUtmConversionMode property changes.
+  \fn void CoordinateConversionController::inputFormatChanged();
+  \brief Signal emitted when the list of \l inputFormat property changes.
  */
 
 /*!
-  \fn void CoordinateConversionController::runConversionChanged();
-  \brief Signal emitted when the \l runConversion property changes.
+  \fn void CoordinateConversionController::captureModeChanged();
+  \brief Signal emitted when the list of \l captureMode property changes.
```

Comment:
We typically don't mention in the signal doc that it's a list property. Is that needed here?

Reply:
(there are three of these that mention `list of` in case you want to change them)